docs/manual/external-tools.tex

```diff
@@ -328,58 +328,61 @@ \section{Gradle\label{gradle}}
 
 \section{IntelliJ IDEA\label{intellij}}
 
-% The following method has been tested with IntelliJ IDEA 15.0.3.
+% The following method has been tested with IntelliJ IDEA 2017.3.4 Community Edition
 
 To run a checker within IntelliJ:
 
-\begin{enumerate}
+Note: Keymaps references are for Mac OS, to find your default keymaps,
```

Comment:
Let's just remove all mentions of keymaps in the instructions since they are platform dependent.